src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php

```diff
@@ -69,6 +69,30 @@ public function getEventTransfersByModifiedColumns(array $eventTransfers, array
         return $validEventTransfers;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\EventEntityTransfer[] $eventTransfers
+     * @param string $columnName
+     *
+     * @return array
+     */
+    public function getEventTransfersOriginalValues(array $eventTransfers, string $columnName): array
+    {
+        if (!$columnName) {
+            return [];
+        }
+
+        $originalValues = [];
+        foreach ($eventTransfers as $eventTransfer) {
+            if (!isset($eventTransfer->getOriginalValues()[$columnName])) {
```

Comment:
Save into local var `$originalValuesOfEvent` first to avoid 2nd same lookup call below.

Reply:
It's not exactly "lookup" is it? it's `O(1)`.

Reply:
instead of method()[] stuff:
```php
$originalValuesOfEvent = $eventTransfer->getOriginalValues();
if (!isset($originalValuesOfEvent[$columnName])) {
    $originalValues[] = $originalValuesOfEvent[$columnName];
}
```
is cleaner

Reply:
I don't think it's much cleaner, and I prefer to keep the class consistent actually as in line 41: https://github.com/spryker/event-behavior/pull/22/files#diff-a20de33cc3683e03af276bd65d3c154bR41

Reply:
I agree with @dereuromark , `method()[]` is not clean, @m7moud we can also fix the other places when we have a cleaner solution